### PR TITLE
Adding a Dockerfile

### PR DESCRIPTION
**Pull request by benclapp**
Multi stage dockerfile, to produce a small image with just the compiled binary and some root certificates. Users will need to mount their `config.json` file, but otherwise it's pretty standard.

I've pushed an image here as well: https://hub.docker.com/r/benclapp/plex_exporter

### CODE REVIEW

**Comment by arnarg** [comment]
Thanks for contributing

**Comment by benclapp** [comment]
Thanks for the rest of the heavy lifting!

**Comment by arnarg** [comment]
I'm just glad someone else thinks it's useful. Is there any metric missing that you'd like to have?

**Comment by benclapp** [comment]
I was going to raise a separate issue for this when I got a chance to debug it, but I'm leaving for holiday for a couple weeks tomorrow so pretty busy atm. 

TLDR, all plex metrics seem to disappear from `/metrics` whenever any user is playing content from the Plex server. Log line below... Guess I could raise the issue now as a place holder, and provide better details when I'm back in a couple weeks.

```
ERRO[0023] Could not get metrics for server "mediaserver"  context=client server=mediaserver
DEBU[0023] Could not get session count: json: cannot unmarshal string into Go struct field User.id of type int  context=client server=mediaserver
```

**Comment by arnarg** [comment]
I see. I actually haven't been running it in "production" yet so I haven't run into issues like that.

I'm sure the culprit is https://github.com/arnarg/plex_exporter/blob/master/plex/api/session.go#L23

I think dropping the whole user field would be best until we have any use for it. 

I'm going on vacation as well so please just file the issue when you get back :)
